starknet_py/net/schemas/rpc/contract.py

```diff
@@ -87,48 +81,68 @@ def make_dataclass(self, data, **kwargs) -> SierraEntryPointsByType:
         return SierraEntryPointsByType(**data)
 
 
+class EntryPointsByTypeSchema(Schema):
+    constructor = fields.List(
+        fields.Nested(EntryPointSchema()), data_key="CONSTRUCTOR", required=True
+    )
+    external = fields.List(
+        fields.Nested(EntryPointSchema()), data_key="EXTERNAL", required=True
+    )
+    l1_handler = fields.List(
+        fields.Nested(EntryPointSchema()), data_key="L1_HANDLER", required=True
+    )
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> EntryPointsByType:
+        return EntryPointsByType(**data)
+
+
 class SierraContractClassSchema(Schema):
+    sierra_program = fields.List(
+        fields.String(), data_key="sierra_program", required=True
```

Comment:
Actually, it would be better to align it with the [spec](https://github.com/starkware-libs/starknet-specs/blob/e5ce8c6f99672d9ddf85a8e1d6ef8e87169d6e8d/api/starknet_api_openrpc.json#L3039) and have it as `Felt`. To maintain consistency with the rest of the code, we should treat felts as ints. This requires making changes in the [SierraContractClass](https://github.com/software-mansion/starknet.py/blob/da1aa2fad8c3066711c6614f2e3140f444f5cbe9/starknet_py/net/client_models.py#L815) and consequently in the [compute_sierra_class_hash](https://github.com/software-mansion/starknet.py/blob/da1aa2fad8c3066711c6614f2e3140f444f5cbe9/starknet_py/hash/sierra_class_hash.py#L34) function as well.
```suggestion
        Felt(), data_key="sierra_program", required=True
```